miner/worker.go

```diff
@@ -240,7 +240,10 @@ func (w *worker) stop() {
 	atomic.StoreInt32(&w.running, 0)
 
 	if istanbul, ok := w.engine.(consensus.Istanbul); ok {
-		istanbul.StopValidating()
+		err := istanbul.StopValidating()
```

Comment:
Is this related to the shutdown of the db?

Reply:
Partially, `StopValidating` could have also returned an error before this change, but this change added another condition where `StopValidating` could return an error.